### PR TITLE
Some final touches in the Migration guide for 2.8

### PR DESCRIPTION
**Pull request by marcospereira**
Also, a bunch of minor fixes at the docs (because I was too tired to do something more significant on the airplane, but not too tired to sleep). :-P

### CODE REVIEW

File: documentation/manual/about/Philosophy.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
 That’s why we created Play 2, a new web framework for a new era.
 
 ## Built for asynchronous programming
 
-Today’s web applications are integrating more concurrent real-time data, so web frameworks need to support a full asynchronous HTTP programming model. Play was initially designed to handle classic web applications with many short-lived requests. But now, the event model is the way to go for persistent connections - through Comet, long-polling and WebSockets.
+Today’s web applications are integrating more concurrent real-time data, so web frameworks need to support a full asynchronous HTTP programming model. We initially design Play to handle classic web applications with many short-lived requests. But now, the event model is the way to go for persistent connections — through Comet, long-polling and WebSockets.
```
Comment (ignasi35) [inline documentation/manual/about/Philosophy.md]:
```suggestion
Today’s web applications are integrating more concurrent real-time data, so web frameworks need to support a full asynchronous HTTP programming model. We initially designed Play to handle classic web applications with many short-lived requests. But now, the event model is the way to go for persistent connections — through Comet, long-polling and WebSockets.
```

File: documentation/manual/about/PlayUserGroups.md [hunk 2]
```diff
@@ -16,29 +12,24 @@ https://www.meetup.com/Play-NYC/
...
+<https://www.meetup.com/PlayFramework-Wien/>
 
 ## Belgium
 
 <https://groups.google.com/forum/#!forum/play-be>
 
 ## Japan
 
-* https://groups.google.com/forum/?fromgroups#!forum/play_ja
+* <https://groups.google.com/forum/?fromgroups#!forum/play_ja>
```
Comment (ignasi35) [inline documentation/manual/about/PlayUserGroups.md]:
```suggestion
<https://groups.google.com/forum/?fromgroups#!forum/play_ja>
```

File: documentation/manual/releases/release27/migration27/Migration27.md [hunk 3]
```diff
@@ -561,7 +561,7 @@ Jackson version was updated from 2.8 to 2.9. The release notes for this version
 
 ### Hibernate Validator updated to 6.0
 
-[Hibernate Validator](http://hibernate.org/validator/) was updated to version 6.0 which is now compatible with [Bean Validation](https://beanvalidation.org/) 2.0. See what is new [here](http://hibernate.org/validator/releases/6.0/#whats-new) or read [this detailed blog post](http://in.relation.to/2017/08/07/and-here-comes-hibernate-validator-60/) about the new version.
+[Hibernate Validator](http://hibernate.org/validator/) was updated to version 6.0 which is now compatible with [Bean Validation](https://beanvalidation.org/) 2.0. See what is new [here](http://hibernate.org/validator/releases/6.0/#whats-new) or read [this detailed blog post](https://in.relation.to2017/08/07/and-here-comes-hibernate-validator-60/) about the new version.
```
Comment (ignasi35) [inline documentation/manual/releases/release27/migration27/Migration27.md]:
```suggestion
[Hibernate Validator](http://hibernate.org/validator/) was updated to version 6.0 which is now compatible with [Bean Validation](https://beanvalidation.org/) 2.0. See what is new [here](http://hibernate.org/validator/releases/6.0/#whats-new) or read [this detailed blog post](https://in.relation.to/2017/08/07/and-here-comes-hibernate-validator-60/) about the new version.
```

A slash disappeared from "in.relation.to/2017/". ;-)

File: documentation/manual/tutorial/Tutorials.md [hunk 4]
```diff
@@ -7,40 +7,39 @@ Tutorials and examples are useful for showing a single application at work, espe
...
+| Streaming Example                          | [Download (zip)](https://example.lightbend.com/v1/download/play-samples-play-scala-streaming-example)      | [Github](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-streaming-example/)      |
+| Compile Time Dependency Injection Example  | [Download (zip)](https://example.lightbend.com/v1/download/play-samples-play-scala-compile-di-example)     | [Github](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-compile-di-example/)     |
+| Dependency Injection using Macwire Example | [Download (zip)](https://example.lightbend.com/v1/download/play-samples-play-scala-macwire-di-example)     | [Github](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-macwire-di-example/)     |
+| Secure Session Example                     | [Download (zip)](https://example.lightbend.com/v1/download/play-samples-play-scala-secure-session-example) | [Github](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-secure-session-example/) |
```
Comment (ignasi35) [inline documentation/manual/tutorial/Tutorials.md]: Oh wow! I have no idea what changed here. I'm assuming it's all good.

File: documentation/manual/releases/release27/migration27/Migration27.md (hunk 3 above)
Comment (marcospereira) [reply documentation/manual/releases/release27/migration27/Migration27.md]: All mighty Ignasi eagle eye!

File: documentation/manual/tutorial/Tutorials.md (hunk 4 above)
Comment (marcospereira) [reply documentation/manual/tutorial/Tutorials.md]: URLs before:

    /v1/download/play-scala-rest-api-example

URL after:

    /v1/download/play-samples-play-scala-rest-api-example

Same applies to Github repo URL.

Comment (marcospereira) [reply documentation/manual/tutorial/Tutorials.md]: By the way, I've caught those by using `validateExternalLinks`. :-)

Comment (ignasi35) [reply documentation/manual/tutorial/Tutorials.md]: > By the way, I've caught those by using validateExternalLinks. :-)

Oh, this is the play repo. I assumed that task was on the Ci build already. Let's PR separately so the `validateExternalLinks` runs at least nightly.

**Pull request merged**